bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/FutureValue.java

```diff
@@ -48,7 +48,7 @@ public FutureValue(Strand strand, CallableUnitCallback callback) {
      }
 
      @Override
-     public String stringValue() {
+     public String stringValue(Strand strand) {
          StringJoiner sj = new StringJoiner(",", "{", "}");
          sj.add("isDone:" + isDone);
```

Comment:
Printing such information in toString is not required IMO. @Kishanthan @sameerajayasoma wdyt ?